src/internal/_pipe.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function _pipe(_f, g) {
+  return {
+    length: _f.length + g.length - 1,
+    value: function() {
+      var fArgs = [];
+      var gArgs = [];
+      var idx = 0;
+      while (idx < arguments.length) {
+        (idx < _f.length ? fArgs : gArgs).push(arguments[idx]);
+        idx += 1;
+      }
+      return g.apply(this, [_f.value.apply(this, fArgs)].concat(gArgs));
+    }
+  };
+};
```

Comment:
easy way to do that would be to use `arr[arr.length]` style and leave `[0]` undefined for `gArgs` until you apply `_f`